Mathlib/Topology/Algebra/IsUniformGroup/Defs.lean

```diff
@@ -31,35 +31,39 @@ noncomputable section
 
 open Uniformity Topology Filter Pointwise
 
-section UniformGroup
+section IsUniformGroup
 
 open Filter Set
 
 variable {α : Type*} {β : Type*}
 
 /-- A uniform group is a group in which multiplication and inversion are uniformly continuous. -/
-class UniformGroup (α : Type*) [UniformSpace α] [Group α] : Prop where
+class IsUniformGroup (α : Type*) [UniformSpace α] [Group α] : Prop where
   uniformContinuous_div : UniformContinuous fun p : α × α => p.1 / p.2
 
+@[deprecated (since := "2025-03-26")] alias UniformGroup := IsUniformGroup
+
 /-- A uniform additive group is an additive group in which addition
   and negation are uniformly continuous. -/
-class UniformAddGroup (α : Type*) [UniformSpace α] [AddGroup α] : Prop where
+class IsUniformAddGroup (α : Type*) [UniformSpace α] [AddGroup α] : Prop where
   uniformContinuous_sub : UniformContinuous fun p : α × α => p.1 - p.2
 
-attribute [to_additive] UniformGroup
+@[deprecated (since := "2025-03-26")] alias UniformAddGroup := IsUniformAddGroup
+
+attribute [to_additive] IsUniformGroup
 
 @[to_additive]
-theorem UniformGroup.mk' {α} [UniformSpace α] [Group α]
+theorem IsUniformGroup.mk' {α} [UniformSpace α] [Group α]
     (h₁ : UniformContinuous fun p : α × α => p.1 * p.2) (h₂ : UniformContinuous fun p : α => p⁻¹) :
-    UniformGroup α :=
+    IsUniformGroup α :=
   ⟨by simpa only [div_eq_mul_inv] using
     h₁.comp (uniformContinuous_fst.prodMk (h₂.comp uniformContinuous_snd))⟩
 
-variable [UniformSpace α] [Group α] [UniformGroup α]
+variable [UniformSpace α] [Group α] [IsUniformGroup α]
 
 @[to_additive]
 theorem uniformContinuous_div : UniformContinuous fun p : α × α => p.1 / p.2 :=
-  UniformGroup.uniformContinuous_div
+  IsUniformGroup.uniformContinuous_div
```

Comment:
Do we want a deprecation here?

Reply:
Here only the proof changed, not any name.